Gemfile

```diff
@@ -2,7 +2,7 @@ source "https://rubygems.org"
 gemspec
 
 group :development, :test do
-  gem "omnibus", github: "opscode/omnibus"
+  gem "omnibus", github: "chef/omnibus", branch: "ksubrama/gcc_investigate"
```

Comment:
Totally doesn't need it for it to execute without bugs here - drop the reference.

There will be issues - but it won't be omnibus-software that's at fault.